man/bootctl.xml

```diff
@@ -526,6 +526,20 @@
         <xi:include href="version-info.xml" xpointer="v253"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--secure-boot-auto-enroll</option></term>
+        <listitem><para>Configure the ESP for secure boot auto-enrollment when invoking the
+        <command>install</command> command. Specifying this option will make <command>bootctl</command>
+        populate the ESP with signed <literal>PK</literal>, <literal>KEK</literal> and <literal>db</literal>
+        EFI signature lists which will be picked up and enrolled by <command>systemd-boot</command> if secure
+        boot is in setup mode and secure boot auto-enrollment is enabled.</para>
+
+        <para>When specifying this option, a certificate and private key have to be provided as well using
+        the <option>--certificate=</option> and <option>--private-key=</option> options.</para>
+
```

Comment:
maybe mention explicitly that the certs are cnverted automatically from DER to ESL and signed.

Reply:
i sense sooner or later people want to provide already prepared ESL via this tool, but i guess it#s fine this way for now

Reply:
that automatic DER → ESL/signing thing is still not in there

Reply:
still not addressed?

Reply:
i.e. something like this: "The PEM encoded X.509 certificate is automatically converted into the EFI Signature List (ESL) as needed and self-signed."

Reply:
@poettering It is addressed, in the previous section

Reply:
where? i see no mention about the conversion, the signing and stuff. I do see you added a reference to ESL to the end of this paragraph, but I think the fact that we convert the key to ESL should be in the first paragraph.

Reply:
> Enabling this option will make <command>bootctl</command> populate the ESP with signed <literal>PK</literal>, <literal>KEK</literal> and <literal>db</literal> signature databases, each containing the given certificate in <literal>DER</literal> format as their only entry.

This covers the conversion and the fact that the ESLs contain a single entry with the certificate no?